projects/schematics/src/migrations/2_0/constructor-deprecations/constructor-deprecation-data.ts

```diff
@@ -33,17 +38,17 @@ import { PAYMENT_METHOD_COMPONENT_MIGRATIONS } from './data/payment-method.compo
 import { PRODUCT_CAROUSEL_COMPONENT_MIGRATION } from './data/product-carousel-component.migration';
 import { PRODUCT_SERVICE_MIGRATION } from './data/product-service.migration';
 import { PROMOTION_SERVICE_MIGRATION } from './data/promotion.service.migration';
+import { REGISTER_COMPONENT_MIGRATIONS } from './data/register.component.migration';
```

Comment:
this migration is not part of this PR

Reply:
yup, but I believe it was just a change of position from prettier since I don't use those migration in my PR

Reply:
got moved from line 41